mailbox/backup/src/test/java/org/apache/james/mailbox/backup/ZipAssert.java

```diff
@@ -127,4 +134,24 @@ public ZipAssert hasNoEntry() {
         return myself;
     }
 
+    public ZipAssert hasSameContentWith(ZipFile anotherZipFile) throws IOException {
+        if (zipFile == null || anotherZipFile == null) {
+            throwAssertionError(shouldEqual(zipFile, anotherZipFile));
+        }
+        ArrayList<ZipArchiveEntry> entries = Collections.list(zipFile.getEntries());
+        ArrayList<ZipArchiveEntry> entriesOfAnother = Collections.list(anotherZipFile.getEntries());
+
+        if (entries.size() != entriesOfAnother.size()) {
+            throwAssertionError(shouldEqual(zipFile, anotherZipFile));
+        }
+
+        for (int i = 0; i < entries.size(); i++) {
+            ZipArchiveEntry entry = entries.get(i);
+            ZipArchiveEntry entryOfAnother = entriesOfAnother.get(i);
+            assertThat(zipFile.getInputStream(entry))
+                .hasSameContentAs(zipFile.getInputStream(entryOfAnother));
```

Comment:
We should have a description that `entry n°36 is not the same" `

We should also display entries name